context/Dockerfile

```diff
@@ -25,10 +25,18 @@ RUN npm run build
 
 FROM tiangolo/uwsgi-nginx-flask:python${PYTHON_MINOR_V}
 
+# Use a constraints file to force pip to use cython<3 at build time to address:
+# "AttributeError: cython_sources" with Cython 3.0.0a10 #601 (comment)
+# https://stackoverflow.com/questions/77490435/attributeerror-cython-sources
+# May be removed after upgrading to PyYaml 6
+# https://github.com/yaml/pyyaml/issues/724
+ARG PIP_CONSTRAINT="cython<3"
+
 COPY requirements.txt /app
 # Legacy resolver is used to avoid issues with a non-pinned subdependency which causes build failures in CI
 # https://github.com/hubmapconsortium/portal-ui/actions/runs/6087458665/job/16516023950
-RUN pip install -r /app/requirements.txt --use-deprecated=legacy-resolver
+RUN echo "cython<3" > /tmp/constraint.txt
```

Comment:
Why do we both `PIP_CONSTRAINT="cython<3"` and `RUN echo "cython<3" > /tmp/constraint.txt`?

Reply:
We don't. That was leftover from an earlier attempt. Thanks for catching that!